app/views/content_items/detailed_guide.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.title %>
+
+<%= render 'govuk_component/breadcrumbs', breadcrumbs: @content_item.breadcrumbs %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: "Guidance",
```

Comment:
It is now:

![image](https://cloud.githubusercontent.com/assets/6908677/14676420/a526cc0c-070d-11e6-994a-7cc63595ebd7.png)